Source/Config/Frames/Items.xml

```diff
@@ -0,0 +1,95 @@
+      <Frame inherits="AuctionatorConfigurationTitleFrame" parentKey="TitleArea">
+        <KeyValues>
+          <KeyValue key="titleText" value="Selling Items" type="string"/>
+          <KeyValue key="subTitleText" value="Options to set undercutting percent or values and duration for items." type="string"/>
+        </KeyValues>
+      </Frame>
+
+      <Frame inherits="AuctionatorConfigurationRadioButtonGroup" parentKey="ItemDurationGroup">
+        <KeyValues>
+          <KeyValue key="groupHeadingText" value="Default Auction Duration" type="string"/>
+        </KeyValues>
+        <Anchors>
+          <Anchor point="TOPLEFT" relativeKey="$parent.TitleArea" relativePoint="BOTTOMLEFT"/>
+        </Anchors>
+        <Frames>
+          <Frame inherits="AuctionatorConfigurationRadioButton">
+            <KeyValues>
+              <KeyValue key="labelText" value="12 Hours" type="string"/>
+              <KeyValue key="value" value="12" type="number"/>
+            </KeyValues>
+          </Frame>
+          <Frame inherits="AuctionatorConfigurationRadioButton">
+            <KeyValues>
+              <KeyValue key="labelText" value="24 Hours" type="string"/>
+              <KeyValue key="value" value="24" type="number"/>
+            </KeyValues>
+          </Frame>
+          <Frame inherits="AuctionatorConfigurationRadioButton">
+            <KeyValues>
+              <KeyValue key="labelText" value="48 Hours" type="string"/>
+              <KeyValue key="value" value="48" type="number"/>
+            </KeyValues>
+          </Frame>
+        </Frames>
+      </Frame>
+
+      <Frame inherits="AuctionatorConfigurationRadioButtonGroup" parentKey="ItemSalesPreference">
+        <KeyValues>
+          <KeyValue key="groupHeadingText" value="Sales Preference" type="string"/>
+        </KeyValues>
+        <Anchors>
+          <Anchor point="TOPLEFT" relativeKey="$parent.ItemDurationGroup" relativePoint="BOTTOMLEFT"/>
+        </Anchors>
+        <Frames>
+          <Frame inherits="AuctionatorConfigurationRadioButton">
+            <!--
+              The values here should be populated by Auctionator.Config.SalesTypes but I don't know
+              how to do that programmatically?
+            -->
+            <KeyValues>
+              <KeyValue key="labelText" value="Percentage" type="string"/>
+              <KeyValue key="value" value="percentage" type="string"/>
+            </KeyValues>
+          </Frame>
+          <Frame inherits="AuctionatorConfigurationRadioButton">
+            <KeyValues>
+              <KeyValue key="labelText" value="Set Value" type="string"/>
+              <KeyValue key="value" value="static" type="string"/>
+            </KeyValues>
+          </Frame>
+        </Frames>
+      </Frame>
+
+      <Frame inherits="AuctionatorConfigurationNumericInput" parentKey="ItemUndercutPercentage" hidden="true">
+        <KeyValues>
+          <KeyValue key="labelText" value="% Undercut percentage" type="string"/>
```

Comment:
Are all the config panels doubled? There's undercut percentage here, but it was the same with value. Why are there 2 identical sets of controls?